### DISCUSSION TITLE
安装文档2.1节

### DISCUSSION THREAD

**Discussion by littleforce163** [apache/doris#7566] [A - General / Q&A] [answered]
export REPOSITORY_URL=https://doris-incubating-repo.bj.bcebos.com/thirdparty
这里下载失败
例如：https://doris-incubating-repo.bj.bcebos.com/thirdparty/openssl-1.0.2k.tar.gz
返回：{"code":"NoSuchBucket","message":"The specified bucket does not exist.","requestId":"6fe28412-142a-4fa1-b420-7a16fd680c23"}
是不是网址错了？

**Comment by BiteTheDDDDt** [comment]
https://github.com/openssl/openssl/archive/OpenSSL_1_1_1m.tar.gz
https://github.com/openssl/openssl/archive/OpenSSL_1_0_2k.tar.gz
可以先在这下

**Comment by littleforce163** [comment]
不止这一个，许许多多呢，希望url能下载全部，谢谢

**Comment by BiteTheDDDDt** [comment, marked as answer]
`export REPOSITORY_URL=https://doris-thirdparty-repo.bj.bcebos.com/thirdparty`
试试这个

**Reply by littleforce163** [reply]
Nice，这个url可以下载，谢谢
